api/v1beta1/placementapi_webhook.go

```diff
@@ -86,7 +92,25 @@ func (r *PlacementAPI) ValidateCreate() error {
 func (r *PlacementAPI) ValidateUpdate(old runtime.Object) error {
```

Comment:
please note that this needs to be called from openstack-operator to meaningfully reject the update. See https://github.com/openstack-k8s-operators/openstack-operator/pull/355

Reply:
Yes. I'm aware of that requirement.

Reply:
I'd work on that integration separately so I'll resolve this conversation now but please let me know if you want me to address anything early within this change.